internal/graph/lookupresources.go

```diff
@@ -55,14 +55,14 @@ func (cl *CursoredLookupResources) LookupResources(
 	reachableContext, cancelReachable := context.WithCancel(newContextForReachable)
 	defer cancelReachable()
 
-	limits, lCtx := newLimitTracker(lookupContext, req.OptionalLimit)
+	limits, _ := newLimitTracker(lookupContext, req.OptionalLimit)
```

Comment:
👍🏻  reaching the limit should cancel RR but not LR. Im surprised we didn't have a test that would causes this to trigger.